internal/database/store.go

```diff
@@ -0,0 +1,88 @@
+package database
+
+import (
+	"cloud.google.com/go/datastore"
+	"context"
+	"errors"
+	"fmt"
+	"github.com/ONSdigital/ras-rm-print-file/pkg"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"time"
+)
+
+type DataStore struct {
+	ctx    context.Context
```

Comment:
Is it idomatic to have `ctx` instead of `context`?  The latter is easier to read at a glance, but I might be wrong

Reply:
its a golang thing. Single letter variables are apparently the standard

Reply:
I wasn't sure on that but I do see it a lot in Golang hence why I've stuck with it. However I've notice I switch between single letter variables and full names. I'll make in more consistent in the next PR.